### DISCUSSION TITLE
Photoprism will not upload large videos - truenas / truecharts -

### DISCUSSION THREAD

**Discussion by canosgmk** [photoprism/photoprism#3016] [Q&A]
Hello, 

using Truenas Scale, PhotoPrism through TrueScale.  

All photos and videos under 20/30secs has no issues uploading and meta data populated. 

Any videos above that short time or above 30mb, like 1.09Gb video will be prepared, goes though the entire process and say ‘complete’ or ‘error’.  Either way, the video doesn’t populated.

help please.

**Comment by graciousgrey** [comment]
You might be running into a size limit. It is configurable using this variable: [PHOTOPRISM_ORIGINALS_LIMIT](https://docs.photoprism.app/getting-started/config-options/)
Unfortunately we cannot tell you what limit is configured by TrueNAS. Feel free to contact their team regarding this.
